marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/entitySearch/saveSavedQueryTest.sjs

```diff
@@ -38,45 +40,150 @@ var saveQuery = JSON.stringify({
   }
 });
 
-function invokeService(saveQuery) {
-  return fn.head(xdmp.invoke(
-      "/data-hub/5/data-services/entitySearch/saveSavedQuery.sjs",
-      {
-        "saveQuery": saveQuery
-      }
-  ));
+function beforeEach() {
+  declareUpdate();
+  xdmp.collectionDelete("http://marklogic.com/data-hub/saved-query");
```

Comment:
Just call "reset-hub()" in data-hub-test-helper.xqy instead. That ensures that everything but the "hub-core-artifact" documents are retained.